lib/inspec/utils/telemetry/collector.rb

```diff
@@ -17,17 +26,20 @@ def add_data_series(data_series)
       @data_series << data_series
     end
 
-    # Is the Telemetry system enabled or disabled?
-    # Always true until we add configuration parsing.
+    # The loaded configuration should have a option to configure
+    # telemetry, if not default to false.
     # @return [True, False]
     def telemetry_enabled?
-      @enabled
+      if @telemetry_toggled_off
+        false
+      else
+        config_telemetry_options.fetch('enable_telemetry', false)
```

Comment:
I usually like my booleans to be positively oriented, because logical statements like this usually throw me for a bit of a De Morgan-y loop mentally. 

Also, if statements that have a return value of true/false in them is usually a sign that the logic can be cleaned up into an or/and expression... but see point #1. 😛

Reply:
Ok. I'll give this one some more thought and see how I can make it better.